packages/js/src/helpers/disableMetaboxTabs.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Disables a metabox tab.
+ *
+ * @param {string} tabId The id of the tab to disable.
+ *
+ * @returns {void}
+ */
+function disable( tabId ) {
+	const tab = document.querySelector( `#${ tabId }` );
+	if ( tab === null ) {
+		return;
+	}
+
+	tab.style.opacity = "0.5";
+	tab.style.pointerEvents = "none";
+	tab.setAttribute( "aria-disabled", "true" );
+
+	if ( tab.classList.contains( "yoast-active-tab" ) ) {
+		tab.classList.remove( "yoast-active-tab" );
+	}
```

Comment:
No need to change, but FYI I think that `classList.remove` with a non-existing classname is safe to call